src/components/BookmarksPanel/BookmarksPanel.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+
+import Bookmark from 'components/Bookmark';
+import EditingBookmark from 'components/Bookmark/EditingBookmark';
+import Icon from 'components/Icon';
+
+import actions from 'actions';
+import selectors from 'selectors';
+
+import './BookmarksPanel.scss';
+
+class BookmarksPanel extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isAdding: false,
+    };
+  }
+
+  static propTypes = {
+    bookmarks: PropTypes.arrayOf(PropTypes.object),
+    addBookmark: PropTypes.func.isRequired,
+    display: PropTypes.string.isRequired,
+    currentPage: PropTypes.number.isRequired,
+    isDisabled: PropTypes.bool,
+  }
+
+  render() {
+    const { isDisabled, display, bookmarks, addBookmark, currentPage, t } = this.props;
+
+    if (isDisabled) {
+      return null;
+    }
+
+    const pageIndexes = Object.keys(bookmarks).map(pageIndex => parseInt(pageIndex, 10));
+    return (
+      <div className="Panel bookmarks-panel" style={{ display }} data-element="bookmarksPanel">
```

Comment:
I kinda want to rename the class name to be `BookmarksPanel`, just to make it consistent with other components. For example we use `NotesPanel` instead of `notes-panel`... Let me know what you think about this.

We can probably remove this kind of class names when implementing the new UI design because customers can just select the `data-element` to customize the css styles and `data-element` is unique. What do you think?

Reply:
changed bookmarks-panel to bookmarksPanel

you mean by using [attribute=value] css selector with data-element?

Reply:
No just the className.

`className="Panel bookmarks-panel"` to `className="Panel BookmarksPanel"`

Reply:
k done